packages/typespec-ts/test/modularUnit/scenarios/operations/pathParam/optionalPath.md

```diff
@@ -0,0 +1,73 @@
+# Should support path parameters
+
+## TypeSpec
+
+This is tsp definition.
+
+```tsp
+op read(@path param?: string): OkResponse;
```

Comment:
does optional path params work in RLC?

Reply:
issue tracked https://github.com/Azure/autorest.typescript/issues/3178